boards/DCM/Inc/App/App_BrakeLight.h

```diff
@@ -2,12 +2,6 @@
 
 #include <stdbool.h>
 
```

Comment:
Did some slight clean up in this PR

- Removed `enum` and replaced it with a simpler boolean
- Cleaned up some unncessary `const`
- Touch up the test files according to the last 2 points